target/linux/ar71xx/image/tp-link.mk

```diff
@@ -669,6 +669,22 @@ define Device/tl-wr1041n-v2
 endef
 TARGET_DEVICES += tl-wr1041n-v2
 
+define Device/tl-wr1043n-v5
+  $(Device/tplink)
+  DEVICE_TITLE := TP-LINK TL-WR1043N v5
+  BOARDNAME := TL-WR1043N-v5
+  DEVICE_PROFILE := TLWR1043
+  TPLINK_HWID :=  0x10430005
```

Comment:
this entry is used by mktplinkfw but You use only uImageArcher and tplink-safeloader